### PR TITLE
[5.0.x] Updated source translation catalogs.

### PR DESCRIPTION
**Pull request by nessita**
This is the result of running:

```
django-admin makemessages -l en
```

### CODE REVIEW

**Comment by carltongibson** [comment]
@claudep and @felixxm have previously tried to reduce the diff here (where it's only line numbers that changed). That's not really documented, I wonder if they might comment (and maybe we can add that to the _How is Django Formed?_ checklist)

**Comment by nessita** [comment]
> @claudep and @felixxm have previously tried to reduce the diff here (where it's only line numbers that changed). That's not really documented, I wonder if they might comment (and maybe we can add that to the _How is Django Formed?_ checklist)

I did wonder about that, and also why some "Translators: ..." messages were removed. I restored the latter but did nothing with the former, though I agree it makes sense to reduce the diff by ignoring files that only have line numbers changed. Will make a new commit and squash all down. Thanks!

File: django/contrib/humanize/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -329,47 +329,47 @@ msgstr[0] ""
...
+#: contrib/humanize/templatetags/humanize.py:274
 #, python-format
 msgctxt "naturaltime-future"
 msgid "%(num)d hour"
 msgid_plural "%(num)d hours"
 msgstr[0] ""
 msgstr[1] ""
 
-#: contrib/humanize/templatetags/humanize.py:219
+#: contrib/humanize/templatetags/humanize.py:277
```
Comment (felixxm) [inline django/contrib/humanize/locale/en/LC_MESSAGES/django.po:372]: I don't see any new strings in `humanize`, please revert.

Comment (nessita) [reply django/contrib/humanize/locale/en/LC_MESSAGES/django.po:372]: I figured it was important for translators to have the updated comment. Is that the case?

Comment (felixxm) [reply django/contrib/humanize/locale/en/LC_MESSAGES/django.po:372]: I don't think it's important, but it's your decision.

**Review by felixxm — COMMENTED** [review]
@nessita Thanks :+1: Can you also update translations for `--domain=djangojs`? There are some errors that we should fix in advance:
```
xgettext: warning: msgid '' is used without plural and with plural.
                   ./views/templates/i18n_catalog.js:52: Here is the occurrence without plural.
                   ./views/templates/i18n_catalog.js:60: Here is the occurrence with plural.
                   Workaround: If the msgid is a sentence, change the wording of the sentence; otherwise, use contexts for disambiguation.
```

**Comment by nessita** [comment]
> @nessita Thanks 👍 Can you also update translations for `--domain=djangojs`? There are some errors that we should fix in advance:
> 
> ```
> xgettext: warning: msgid '' is used without plural and with plural.
>                    ./views/templates/i18n_catalog.js:52: Here is the occurrence without plural.
>                    ./views/templates/i18n_catalog.js:60: Here is the occurrence with plural.
>                    Workaround: If the msgid is a sentence, change the wording of the sentence; otherwise, use contexts for disambiguation.
> ```

On it!

**Comment by nessita** [comment]
> @nessita Thanks 👍 Can you also update translations for `--domain=djangojs`? There are some errors that we should fix in advance:
> 
> ```
> xgettext: warning: msgid '' is used without plural and with plural.
>                    ./views/templates/i18n_catalog.js:52: Here is the occurrence without plural.
>                    ./views/templates/i18n_catalog.js:60: Here is the occurrence with plural.
>                    Workaround: If the msgid is a sentence, change the wording of the sentence; otherwise, use contexts for disambiguation.
> ```

I'm very lost in this one. As far as I have been able to debug, these errors com from these two lines from the django/views/templates/i18n_catalog.js file:

```js
let value = django.gettext(context + '\x04' + msgid);
```

and

```js
let value = django.ngettext(context + '\x04' + singular, context + '\x04' + plural, count);
```

This file [django/views/templates/i18n_catalog.js](https://github.com/ngnpope/django/blob/main/django/views/templates/i18n_catalog.js) was upgraded to a "full javascript" file in PR #16562: before this work, the javascript for the `i18n_catalog` was inline in the `i18n.py` module. So my wild and uneducated guess is that gettext is picking up the string above for translations and complaining because the generated `pot` looks like this:

``` 
# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-09-18 14:48-0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"

#: views/templates/i18n_catalog.js:52 views/templates/i18n_catalog.js:60
msgid ""
msgid_plural ""
msgstr[0] ""
msgstr[1] ""
```

@felixxm @claudep @ngnpope would you have any idea about how to fix this? Could the alpha release be progressed without the JS strings today? I can backport those once we have a fix for this `makemessages` error. Thanks!

**Comment by ngnpope** [comment]
Hey @nessita.

So prior to 8eef22dfed2 the content of `django/views/templates/i18n_catalog.js` was stored as a raw string in a variable named `js_catalog_template` in `django/views/i18n.py`. As such, the content would not have been translated. The issue here is that it now shows up incorrectly as needing translations due to the presence of `gettext`, `ngettext`, etc. in the file.

The fix here would be to exclude this file from translation. I'm not sure how we go about that off the top of my head.

**Comment by nessita** [comment]
> Hey @nessita.
> 
> So prior to [8eef22d](https://github.com/django/django/commit/8eef22dfed2d53df0da10c0090d9cb04f66efb15) the content of `django/views/templates/i18n_catalog.js` was stored as a raw string in a variable named `js_catalog_template` in `django/views/i18n.py`. As such, the content would not have been translated. The issue here is that it now shows up incorrectly as needing translations due to the presence of `gettext`, `ngettext`, etc. in the file.
> 
> The fix here would be to exclude this file from translation. I'm not sure how we go about that off the top of my head.

I agree, my last commit is the result of running:

```
django-admin makemessages -l en --domain=djangojs -i views/templates/i18n_catalog.js 
```

But we'd need to work out a way to having a consistent procedure to generate translations that is not confusing for others.

**Review by felixxm — APPROVED** [review]
We can proceed with this for now :+1:

**Pull request merged**